wavelength-plugins/tasks.js

```diff
@@ -0,0 +1,97 @@
+/*********************************
+ * Tasks (To-Do/Jobs) Plug-in		*
+ * Created for Pokemon Showdown	*
+ * Created by Insist					*
+ ********************************/
+
+"use strict";
+
+function isDev(user) {
+	if (!user) return;
+	if (typeof user === "object") user = user.userid;
+	let dev = Db.devs.get(toId(user));
+	if (dev === 1) return true;
+	return false;
+}
+
+function alertDevs(message) {
+	let developers = Db.devs.keys();
+	for (let u in developers) {
+		if (!Users(developers[u]) || !Users(developers[u]).connected) continue;
+		Users(developers[u]).send(`|pm|~Developer Alert|~|/raw ${message}`);
+	}
+	if (Rooms(`development`)) Rooms(`development`).add(`|c|~Developer Alert|/raw ${message}`).update();
+}
+
+exports.commands = {
+	jobs: "tasks",
+	job: "tasks",
+	todo: "tasks",
+	task: "tasks",
+	tasks: {
+		new: "add",
+		issue: "add",
+		add: function (target, room, user) {
+			if (!isDev(user.userid) && !this.can("bypassall")) return false;
+			let [issue, priority, ...description] = target.split(",").map(p => p.trim());
+			let task = Db.tasks.get("development");
+			if (!issue || !priority || !description) return this.parse("/taskshelp");
+			if (task.issues[toId(issue)]) return this.errorReply(`This issue title already exists.`);
+			if (issue.length < 1 || issue.length > 30) return this.errorReply(`The issue title should not exceed 30 characters long. Feel free to continue in the description.`);
+			if (description.length < 1 || description.length > 100) return this.errorReply(`The description should not exceed 100 characters long.`);
+			if (isNaN(priority) || priority > 6 || priority < 1) return this.errorReply(`The priority should be an integer between 1-6; 1 being the highest priority.`);
+			task.issues[toId(issue)] = {"id": toId(issue), "issue": issue, "description": description, "employer": user.userid, "priority": priority};
```

Comment:
Thanks to ES6/ES7, you can use this for when keys have the same name as variables:
```js
tasks.issues[id] = {id, issue, description, employer: user.userid, priority};
```